<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogregDialog</class>
 <widget class="QDialog" name="LogregDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #101010;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QWidget" name="SignInPage">
    <widget class="QLineEdit" name="usernameLine">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>0</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 20px;
padding-right: 20px;
border-radius: 20px;
background: #333;
color: #fefefe;</string>
     </property>
     <property name="placeholderText">
      <string>Логин</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordLine">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left: 20px;
padding-right: 20px;
border-radius: 20px;
background: #333;
color: #fefefe;</string>
     </property>
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLabel" name="Loader">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>loader</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fefefe;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="RegistrationPage"/>
  </widget>
  <widget class="QPushButton" name="PushButton_SignIn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Padding: 1px;
Border-radius: 5px;
Background: #393;
Color: #fefefe;</string>
   </property>
   <property name="text">
    <string>Вход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
